Gruntfile.js

```diff
@@ -20,9 +20,14 @@
                 
        // Task-specific options go here. 
      },
+  //   font_awesome_svg: {
```

Comment:
okej qetash i fshij pra kom mendu qe herdo kurdo mu qas en to dikur mi dit meniher shka jon